Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -175,7 +114,9 @@ For more information on the rest of the toolkit, please check the following guid
 |  [![App Bar](../External/ReadMeImages/AppBar/MRTK_AppBar_Main.png)](README_AppBar.md) [App Bar](README_AppBar.md) | [![Pointers](../External/ReadMeImages/Pointers/MRTK_Pointer_Main.png)](README_Pointers.md) [Pointers](README_Pointers.md) | [![Fingertip Visualization](../External/ReadMeImages/Fingertip/MRTK_FingertipVisualization_Main.png)](README_FingertipVisualization.md) [Fingertip Visualization](README_FingertipVisualization.md) |
 | UI for Bounding Box's manual activation | Learn about various types of pointers | Visual affordance on the fingertip which improves the confidence for the direct interaction |
 
-## Example Scene
-You can find various types of interactions and UI controls in this example scene.
+# Upgrading from the HoloToolkit (HTK/MRTK v1)
+
+There is not a direct upgrade path from the HoloToolkit to Mixed Reality Toolkit v2 due to the rebuilt framework.  However, it is possible to import the MRTK into your HoloToolkit project and start work migrating your implementation if you wish.
```

Comment:
>implementation if you wish. [](start = 218, length = 27)

We should try and phrase this more of a strong suggestion, rather than a soft option for them. And offer to post an issue if they encounter issues, so they don't feel completely left out in the rain with the upgrade.